### PR TITLE
Polly24 restruct

### PR DESCRIPTION
**Pull request by ulysses78**
implementation of 24h plots, including concatination of level0-files, before processing with picasso.

### CODE REVIEW

File: lib/interface/picassoProcV3.m
Comment (HolgerPollyNet) [inline lib/interface/picassoProcV3.m:1]: would it be wise to clan up before merging or do it afterwards? E.g. a lot of lines are commented out, but maybe not needed anymore at all?

File: lib/visualization/pypolly_display_ATT_BETA.py
Comment (HolgerPollyNet) [inline lib/visualization/pypolly_display_ATT_BETA.py:1]: Please also make it Picasso Version 4.0 now ;-)

File: lib/visualization/pypolly_readout.py [hunk 1]
```diff
@@ -0,0 +1,938 @@
...
+        global_attr[nc_attr] = att_value
+    ## get variable attributes from nc-file
+    var_attr_method = {}
+    var_attr_OL355 = {}
+    var_attr_OL355d = {}
+    var_attr_OL532 = {}
+    var_attr_OL532d = {}
+    var = ["method","overlap355","overlap355Defaults","overlap532","overlap532Defaults"]
+    
+#    for v_count,var_name in enumerate(var):
```
Comment (HolgerPollyNet) [inline lib/visualization/pypolly_readout.py:879]: are the comments still needed? If yes ok, if not, maybe time to clean?

**Review by HolgerPollyNet — APPROVED** [review]
In general it can be merged (as it is working, and never change a working....), but we should clean up soon and delete not needed lines...

Please also make it Version 4.0 as it is a new Zeitrechnung

**Comment by ulysses78** [comment]
I deleted most of the commented out lines and changed the picasso version from 3.6 to 4.0.

**Pull request merged**